packages/twenty-server/src/modules/match-participant/match-participant.service.ts

```diff
@@ -52,7 +52,11 @@ export class MatchParticipantService<
 
     const participantIds = participants.map((participant) => participant.id);
     const uniqueParticipantsHandles = [
-      ...new Set(participants.map((participant) => participant.handle)),
+      ...new Set(
+	      participants
+	      .map((participant) => participant.handle)
+	      .filter((handle) => handle)
```

Comment:
Is an empty string considered empty ? In that case we should use isNonEmptyString guard to make it clear here.